docs/migration/v8-to-v9.md

```diff
@@ -50,40 +45,55 @@ Support for the following frameworks and library versions are dropped:
 
 ### TypeScript Version Policy
 
-In preparation for the OpenTelemetry SDK v2, which will raise the minimum required TypeScript version, the minimum required TypeScript version is increased to version `5.0.4`.
+In preparation for v2 of the OpenTelemetry SDK, which will raise the minimum required TypeScript version, the minimum required TypeScript version is increased to version `5.0.4`.
 
 Additionally, like the OpenTelemetry SDK, the Sentry JavaScript SDK will follow [DefinitelyType's version support policy](https://github.com/DefinitelyTyped/DefinitelyTyped#support-window) which has a support time frame of 2 years for any released version of TypeScript.
 
-Older Typescript versions _may_ still work, but we will not test them anymore and no more guarantees apply.
+Older Typescript versions _may_ continue to be compatible, but no guarantees apply.
 
 ## 2. Behavior Changes
 
 ### `@sentry/core` / All SDKs
 
-- If you use the optional `captureConsoleIntegration` and set `attachStackTrace: true` in your `Sentry.init` call, console messages will no longer be marked as unhandled (i.e. `handled: false`) but as handled (i.e. `handled: true`). If you want to keep sending them as unhandled, configure the `handled` option when adding the integration:
+- Passing `undefined` as a `tracesSampleRate` option value will now be treated the same as if the option was not defined at all.
+  In previous versions, it was checked whether `tracesSampleRate` or `traceSampler` are _defined_ at all, in `Sentry.init()` to determine whether tracing should be enabled.
+  Passing `undefined` to `tracesSampleRate` previously would enable tracing without the performance monitoring aspect (=tracing being enabled, though no spans would be started).
+  If you want to preserve having tracing enabled without creating spans, set `tracesSampleRate: 0`:
```

Comment:
m: Setting tracesSampleRate:0 doesn't enable TwP. Not setting the rate at all or setting it to `undefined` should enable TwP. https://develop.sentry.dev/sdk/telemetry/traces/tracing-without-performance/#configuration

setting a rate of 0 also forces downstream SDKs to always sample negatively.

Reply:
https://github.com/getsentry/sentry-javascript/pull/15287/commits/ceb88b0f4593a0c93669c5c5c342fca55fd55b24